test/test_packages/TestFailure/test/runtests.jl

```diff
@@ -0,0 +1,7 @@
+using TestFailure
+
+if haskey(ENV, "TEST_SIGNAL")
+    run(`kill -s $(ENV["TEST_SIGNAL"]) $(getpid())`)
```

Comment:
I'm pretty sure this will fail on Windows. I can disable this specific test on Windows if that is the case

Reply:
```
kill: 328: No such process
ERROR: LoadError: failed process: Process(`kill -s KILL 328`, ProcessExited(1)) [1]
Stacktrace:
```
I'll disable the signal tests on Windows